README.md

```diff
@@ -33,541 +30,112 @@ This cookbook requires Chef 12.1+.
-
-## Recipes
-
-### default
-
-Installs the rbenv gem and initializes Chef to use the Lightweight Resources and Providers ([LWRPs][lwrp]).
-
-Use this recipe explicitly if you only want access to the LWRPs provided.
-
-### system_install
-
-Installs the rbenv codebase system-wide (that is, into `/usr/local/rbenv`). This recipe includes _default_.
-
-Use this recipe by itself if you want rbenv installed system-wide but want to handle installing Rubies, invoking LWRPs, etc..
-
-### system
-
-Installs the rbenv codebase system-wide (that is, into `/usr/local/rbenv`) and installs rubies driven off attribute metadata. This recipe includes _default_ and _system_install_.
-
-Use this recipe by itself if you want rbenv installed system-wide with rubies installed.
-
-### user_install
-
-Installs the rbenv codebase for a list of users (selected from the `node['rbenv']['user_installs']` hash). This recipe includes _default_.
-
-Use this recipe by itself if you want rbenv installed for specific users in isolation but want each user to handle installing Rubies, invoking LWRPs, etc.
-
-### user
-
-Installs the rbenv codebase for a list of users (selected from the `node['rbenv']['user_installs']` hash) and installs rubies driven off attribte metadata. This recipe includes _default_ and _user_install_.
-
-Use this recipe by itself if you want rbenv installed for specific users in isolation with rubies installed.
-
-## Attributes
-
-### git_url
-
-The Git URL which is used to install rbenv.
-
-The default is `"git://github.com/sstephenson/rbenv.git"`.
-
-### git_ref
-
-A specific Git branch/tag/reference to use when installing rbenv. For example, to pin rbenv to a specific release:
-
-```ruby
-node.default['rbenv']['git_ref'] = "v0.2.1"
-```
-
-The default is `"v0.4.0"`.
-
-### upgrade
-
-Determines how to handle installing updates to the rbenv. There are currently 2 valid values:
-
-- `"none"`, `false`, or `nil`: will not update rbenv and leave it in its current state.
-- `"sync"` or `true`: updates rbenv to the version specified by the `git_ref` attribute or the head of the master branch by default.
-
-The default is `"none"`.
-
-### root_path
-
-The path prefix to rbenv in a system-wide installation.
-
-The default is `"/usr/local/rbenv"`.
-
-### patch_url
-
-A url to a patch file for your ruby install.
-
-The default is `nil`.
-
-### patch_file
-
-A path to a patch file for your ruby install.
+# Usage
 
-The default is `nil`.
+## Gem
 
-### rubies
-
-A list of additional system-wide rubies to be built and installed using the [ruby_build cookbook][ruby_build_cb]. You **must** include `recipe[ruby_build]` in your run_list for the `rbenv_ruby` LWRP to work properly. For example:
+Used to install a gem into the selected rbenv environment.
 
 ```ruby
-node.default['rbenv']['rubies'] = [ "1.9.3-p0", "jruby-1.6.5" ]
+rbenv_gem 'gem_name' do
```

Comment:
These samples seem kinda confusing and I think a lot of people are going to try to use the commas in them. Also most people have no idea what a name_property is